rusk-recovery/src/keys.rs

```diff
@@ -47,6 +56,16 @@ static PUB_PARAMS: Lazy<PublicParameters> = Lazy::new(|| {
     }
 });
 
+#[tokio::main]
+async fn fetch_pp() -> Result<Vec<u8>, Box<dyn std::error::Error>> {
+    let response = reqwest::get("https://dusk-infra.ams3.digitaloceanspaces.com/trusted-setup/dusk-trusted-setup")
```

Comment:
Should we make this an environment variable?

Reply:
done